RocketChat/Rocket.Chat
# chore: Disable omnichannel endpoints and widget serving code with a new setting

## Proposed changes (including videos or screenshots)



this disables omnichannel entirely from the backend, removing the widget serving code, and the omnichannel specific endpoints, this breaks custom integration when the setting is disabled, as well as custom apps.



## Issue(s)

OC-362
## Steps to test or reproduce

go to the omnichannel settings page, and check widget is disabled option, if disabled, no endpoint for omnichannel should work, and after a restart not even the widget, re-enabling enables the endpoints and the widget right away, without the need for a restart.
## Further comments
